test/junit/scala/collection/mutable/ArrayBufferTest.scala

```diff
@@ -289,4 +289,12 @@ class ArrayBufferTest {
 
     assertEquals(0, buffer.size)
   }
+
+  @Test
+  def testSortInPlace: Unit = {
+    val buffer = ArrayBuffer(3, 2, 1)
+    buffer.sortInPlace()
+
+    assertEquals(ArrayBuffer(1, 2, 3), buffer)
+  }
```

Comment:
OK, now I have my answer :smile:

Reply:
OK, I will squash the commits (a bit messy, with a merge in between...) so you can revise. 
For the future: I intentionally did not force rebase the pushed branch (as advised in the contributor's guide). Should I force rebase in the future for the sake of a cleaner history?

Reply:
It depends. For large PRs it’s simpler to not squash and force push every time so that we can review the changes since the last time. For short PRs it’s OK to squash and force push each time.

Reply:
Commits squashed.